CoastalResilienceNetwork/GeositeFramework
# Downgrade jQuery scrollbar library

* Version 3.1.3 of this library (upgraded in d1969e6) causes a JS error
when the site is loaded from within an iframe. The release notes for
3.1.3 mention that a cross-domain iframe issue was fixed by this release,
but it appears that a different issue was introduced. To allow
the site to function, we downgrade to a version of the library
that works in an iframe.

**Testing**
- Create a basic HTML file that has your local site embedded as an iframe. Something like:
```html
<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
    </head>
    <body>
        <iframe width="500" height="400" src="http://cthom-windev-vm:5001/"></iframe>
    </body>
</html>
```
- Run a local webserver to serve this file and access the site.
- Verify the site loads without any JS errors.

Connects to #623
